h/api/test/views_test.py

```diff
@@ -218,6 +219,16 @@ def test_create_returns_render(search_lib):
 read_fixtures = pytest.mark.usefixtures('search_lib', 'AnnotationEvent')
 
 
+@ read_fixtures
+def test_read_404s_for_group_annotations_if_groups_feature_is_off():
```

Comment:
From a client POV I would prefer more descriptive error codes if a feature is not enabled. I think that is a convention to discuss separately though.